lib/Auth/Source/IRMA.php

```diff
@@ -145,7 +141,7 @@ public static function handleLogin($authStateId, $irma_result) {
      *
      * On a successful login, this function should return the users attributes. On failure,
      * it should throw an exception. If the error was due to invalid IRMA credentials,
-     * a SimpleSAML_Error_Error('IRMA_INVALIDCREDENTIALS') should be thrown.
+     * a SimpleSAML_Error_Error('RESPONSESTATUSNOSUCCESS') should be thrown.
```

Comment:
What is the rationale here for wanting to switch to generic errors instead of the IRMA specific ones?